### ISSUE TITLE
Error when running an empty query

### ISSUE THREAD

**Issue by dregad** [mantisbt-plugins/Query#11]
If you try to run an empty query, a DB error is triggered

```
APPLICATION ERROR 401
Database query failed. Error received from database was #0: for the query: .
```

Additionally, this triggers a deprecation warning
```
DEPRECATED: 'html_entity_decode(): Passing null to parameter #1 ($string) of type string is deprecated' in '.\Query\pages\exec_query.php' line 65
```

I suggest to disable the *Run* button if query is not valid, or at least display a more meaningful and user-friendly message.

**Comment by dregad** [comment]
Found while checking https://github.com/mantisbt-plugins/Query/pull/4#issuecomment-2111660815

**Comment by ghost** [comment]
Fixed in 2.18.
This deprecation warning is on passing a null parameter which now is been taking care off.
